Source/Fuse.Common/Internal/MiniList.uno

```diff
@@ -11,16 +20,26 @@ namespace Fuse.Internal
 	struct MiniList<T> : IList<T> where T : class
 	{
 		object _list;
+		ObjectList<T> AsList { get { return (ObjectList<T>)_list; } }
+		
+		MiniListMode _mode = MiniListMode.Empty;
```

Comment:
It was never the goal to reduce total memory size. It's meant to efficiently handle the cases where 0 and 1 children are common, with an ever-so-slight performance loss for the 2+ cases. The efficiency is primarily in allocation overhead, not necessarily size, though in the 0 and 1 case it is still smaller than a List object. Don't forget it is a `struct` meaning the 0 and 1 case are supported without any allocation whatsoever.